simapp/testutil_network_test.go

```diff
@@ -1,27 +1,63 @@
 package simapp_test
 
 import (
+	"math/rand"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/suite"
 
+	sdkmath "cosmossdk.io/math"
 	"cosmossdk.io/simapp"
 
+	cmtcfg "github.com/cometbft/cometbft/config"
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
 	"github.com/cosmos/cosmos-sdk/testutil/network"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	authsim "github.com/cosmos/cosmos-sdk/x/auth/simulation"
+	banksim "github.com/cosmos/cosmos-sdk/x/bank/simulation"
+	xsim "github.com/cosmos/cosmos-sdk/x/simulation"
 )
 
 type IntegrationTestSuite struct {
 	suite.Suite
 
+	cfg      network.Config
+	v0app    *simapp.SimApp
+	rand     *rand.Rand
+	accounts []simtypes.Account
+
 	network *network.Network
 }
 
 func (s *IntegrationTestSuite) SetupSuite() {
 	s.T().Log("setting up integration test suite")
 
+	s.rand = rand.New(rand.NewSource(42))
```

Comment:
Let's add a comment here
```go
// Using a hardcoded seed for predictability.
// TODO(@elias-orijtech): allow the seed to be passed in.
```

Reply:
For AppHash tests, the seed has to be hardcoded. I prefer to leave this for when MockComet support is added.